scripts/repl.js

```diff
@@ -284,7 +371,34 @@
   /*
    * Initialize the REPL
    */
-  var repl = new REPL();
+  var initialQuery = UriUtils.parseQuery();
+  if (initialQuery && initialQuery.short) {
```

Comment:
Yes, I'm leaning towards that being the right thing to do, but I also don't want to break existing URLs.